libraries/adaptive-expressions-ie11/webpack.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+
+module.exports = (env, argv) => {
+    return {
+        mode: 'none',
+        entry: ['core-js/modules/es.array.find', path.resolve(__dirname, './src/index.ts')],
```

Comment:
Now that this is a separate package let's put the polyfill in the entry point directly.

Reply:
Actualy we dont need that polyfill now as that was added because adaptive expression library was using Array.find before.However, now we are passing adaptive-expression also to babel which has core js built in as usage, so it will autmatically get added. 
Nice catch :-)